api/postgres/postgres.go

```diff
@@ -86,13 +86,13 @@ func (u *UserProfileDB) CreateUserProfile(request *CreateUserDatabaseRequest) *C
 
 	resp := &CreateUserDatabaseResponse{Message: "Successfully added user", Error: nil}
 
-	query := "INSERT INTO user_profile (id, username, displayname) VALUES ($1, $2, $3) RETURNING id, username, displayname"
+	query := "INSERT INTO profile_details (id, username, displayname) VALUES ($1, $2, $3) RETURNING id, username, displayname"
```

Comment:
should update this insert statement to use `TBL_USER` instead of hard-coding the name.